reference/append.rst

```diff
@@ -157,7 +157,7 @@ Citus assigns a unique shard id to each new shard and all its replicas have the
 By default, the \\copy command depends on two configuration parameters for its behavior. These are called citus.shard_max_size and citus.shard_replication_factor.
 
 (1) **citus.shard_max_size :-** This parameter determines the maximum size of a shard created using \\copy, and defaults to 1 GB. If the file is larger than this parameter, \\copy will break it up into multiple shards.
-(2) **citus.shard_replication_factor :-** This parameter determines the number of nodes each shard gets replicated to, and defaults to two. The ideal value for this parameter depends on the size of the cluster and rate of node failure. For example, you may want to increase the replication factor if you run large clusters and observe node failures on a more frequent basis.
+(2) **citus.shard_replication_factor :-** This parameter determines the number of nodes each shard gets replicated to, and defaults to one. The ideal value for this parameter depends on the size of the cluster and rate of node failure. For example, you may want to increase the replication factor if you run large clusters and observe node failures on a more frequent basis.
```

Comment:
>For example, you may want to increase the replication factor if you run large clusters and observe node failures on a more frequent basis.

Should we modify this statement to say something like "Set it to 2 if you want Citus to replicate data automatically and provide fault tolerance. You may want to increase the factor even higher if you run large clusters ... "?